backend/YouTubeV2/Controllers/DonationController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.ComponentModel.DataAnnotations;
+using YouTubeV2.Api.Attributes;
+using YouTubeV2.Application.Model;
+using YouTubeV2.Application.Services;
+
+namespace YouTubeV2.Api.Controllers
+{
+    [ApiController]
+    [Route("donate")]
+    public class DonationController : IdentityControllerBase
+    {
+        private readonly IDonationService _donationService;
+
+        public DonationController(IDonationService donationService) 
+        {
+            _donationService = donationService;
+        }
+
+        [HttpPost("send")]
+        [Roles(Role.Simple, Role.Administrator)]
+        public async Task<ActionResult> SendDontaionAsync([FromQuery][Required] Guid id, [FromQuery][Required] decimal amount)
+        {
+            string? senderId = GetUserId();
+            if (senderId is null) return Forbid();
```

Comment:
Accoring to my version of specification this can only be Ok or bad request
![image](https://github.com/IO2-2023-JB/11/assets/95054664/33d3b967-0293-445a-91ec-254df4ab42d7)
Same with other method